test/regression/cases/quality_gate_logs/experiment.yaml

```diff
@@ -0,0 +1,38 @@
+optimization_goal: egress_throughput
```

Comment:
would CPU be more useful here? The assertion on "no lost bytes" seems sufficient to ensure data is flowing through

Reply:
Ah or memory, yes. I think the expectation here is we'll adequately keep up.

Reply:
Adjusted to CPU in [9be4e06](https://github.com/DataDog/datadog-agent/pull/31326/commits/9be4e06f2aba2b92ce613accd70d78b1ddfef5db).